src/mots/cli.py

```diff
@@ -263,6 +290,13 @@ def main():
     init_logging(debug=args.debug)
 
     if hasattr(args, "func"):
+        if settings.CHECK_FOR_UPDATES:
+            args.include_dev_releases = settings.CHECK_DEV_RELEASES
+            try:
+                check_for_updates(args)
```

Comment:
I would write the update checking logic as it's own function that is called by the command function, and then call that here instead of running the command function itself.